python/ray/data/dataset.py

```diff
@@ -312,16 +315,23 @@ def map_batches(
                 tasks, or ActorPoolStrategy(min, max) to use an autoscaling actor pool.
             batch_format: Specify "native" to use the native block format (promotes
                 tables to Pandas and tensors to NumPy), "pandas" to select
-                ``pandas.DataFrame``, or "pyarrow" to select `pyarrow.Table``.
+                ``pandas.DataFrame``, "pyarrow" to select ``pyarrow.Table``, or "numpy"
+                to select ``numpy.ndarray``. Default is "native".
```

Comment:
Also the new Dict[str, ndarray]?
The batch_format and batch_type not being 1:1 feels more complexity added. Ideally, if we can unify on just one type (maybe Dict[str, ndarray]) would be great.

Reply:
>Also the new Dict[str, ndarray]?

Ah good catch, I'll add that!

>Ideally, if we can unify on just one type (maybe Dict[str, ndarray]) would be great.

We need the `ndarray` format for representing pure tensors, and the `Dict[str, ndarray]` for representing tables, so I don't think that we're going to be able to get around that.